instagram-replica/instagram-backend
# Feature/services enhancements

### Changes: 
- Rename all camel case package names to underscore 
- Add HTTP server to all services
- Add static config file and `Settings` class for all the servers 
- Add a separate config file for RabbitMQ connection 
- Add a way to remotely shutdown services 

### Testing steps: 
- Run any server 
- Send an HTTP request to this route `/info`, you should get the settings corresponding to this server back
- Send an HTTP request to this route `/shutdown`, the server will shut down and will no longer function
